website/docs/r/cis_healthcheck.html.markdown

```diff
@@ -43,6 +43,7 @@ The following arguments are supported:
 - `allow_insecure`-(Optional,bool) Do not validate the certificate when healthcheck use HTTPS.
 - `description` - (Optional,string) Free text description.The Default value is false.
 - `port` - (Optional,int) The TCP port to use for the health check.
+- `header` - (Optional,string) The health check header.
```

Comment:
add elements of header `header` and `values` in docs

Reply:
done